File: MODUL 3/PRAK3/PRAK305-2410817210025-Helga.py
s = int(input());

d = int(s / 86400)  
s %= 86400

h = int(s / 3600) 
s %= 3600


m = int(s / 60)  
s %= 60

if d > 0:
    print(f"{d} hari {h:02}:{m:02}:{s:02}");
else:
    print(f"{h:02}:{m:02}:{s:02}");

File: MODUL 3/PRAK 3/PRAK305-2410817210025-Helga.c
#include <stdio.h>
int main(){
    int s, d, h, m;

    scanf("%d", &s);

    d = s / 86400;
    s %= 86400;

    h = s / 3600;
    s %= 3600;

    m = s / 60;
    s %= 60;

    if(d > 0){
        printf("%d hari %02d:%02d:%02d", d, h, m, s);
    }
    else{
        printf("%02d:%02d:%02d", h, m, s);
    }
}